src/ktopt_planning_context.cpp

```diff
@@ -52,21 +70,65 @@ void KTOptPlanningContext::solve(planning_interface::MotionPlanResponse& res)
   }
 
   // compile into a Kinematic Trajectory Optimization problem
-  auto trajopt = KinematicTrajectoryOptimization(plant_->num_positions(), 10);
+  auto trajopt = KinematicTrajectoryOptimization(
+    plant_->num_positions(),
+    params_.control_points);
+  auto& prog = trajopt.get_mutable_prog();
+
+  // Costs
+  trajopt.AddDurationCost(1.0);
+  trajopt.AddPathLengthCost(1.0);
+  // TODO: Adds quadratic cost
```

Comment:
I'd propose to create an issue for every todo and add a code comment that referes to it. For example:
```
  // TODO(#ISSUENUMBER): Adds quadratic cost
```